secure-embedded-cache/README.md

```diff
@@ -33,53 +35,69 @@ Setup
     mvn clean package -DskipTests
 
 2. To run the install and configuration commands, ensure that the local EAP server is running in Standalone mode
-3. To deploy the __security-domain__ that will be used for Authentication, run the command
+
+3. To configure the keycloak subsystem in EAP, run this command:
 
     mvn wildfly:execute-commands
 
-4. Since we will be using __application-user.properties__ and __application-roles.properties__ files that come with a standard EAP server installation at path: __$JBOSS_HOME/standalone/configuration__, run the following commands from the bin folder of the server installation
+4. Download Red Hat Single Sign-On (RH-SSO) server and Client Adapter for JBoss EAP 7 from Red Hat's Customer portal.
+
+5. Install the Client Adapter in EAP by unpacking the Client Adapter zip file into the root directory of EAP
+   and running the following command:
+
+   ${JBOSS_HOME}/jboss-cli.sh --file=adapter-install-offline.cli
+
+6. Start RH-SSO/Keycloak server with port offset:
+
+    ${KEYCLOAK_HOME}/bin/standalone.sh -Djboss.socket.binding.port-offset=100
```

Comment:
I think it might be easier to use Keycloak docker image.

Reply:
Alternatively, Keycloak server behaves the same way as EAP/WF. You can deploy an app on top of that (as the matter of fact we do this in our quickstarts/examples).

Reply:
hi @slaskawi , I was trying to find an example quickstart which is deployed to Keycloak and not WildFly but they seemed to be designed to use both Keycloak and WildFly (this is described in the top-level readme: https://github.com/keycloak/keycloak-quickstarts). I tried deploying to Keycloak directly but I got e.g. this one: https://gist.github.com/mgencur/9c42fbd84b0a7e66d647dca153a3847e
Do you have any example where only Keycloak is used?
Perhaps I'd avoid Docker for this quickstart.

Reply:
Yeah, I believe all examples are deployed only on Keycloak: https://github.com/keycloak/keycloak/tree/master/examples

Reply:
OK, thanks. And (AFAIR) all the quickstarts use WildFly and a separate Keycloak server. I'm not sure what is more common, obviously both options are possible.